usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/CleanOrCreatePath.java

```diff
@@ -25,17 +25,33 @@
 import java.nio.file.Path;
 import java.util.Comparator;
 
+/**
+ * Use case to create a path. This use case is triggered after parsing the rows of a specific file. The resultant path
```

Comment:
👍 This is a really good Javadoc. It gives you a lot of context about what this class does. Because this can be a destructive operation, I would suggest adding the note you have later in `execute()` to this Javadoc too:

>If the target location is not empty, all files at target are deleted.